test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -18,6 +18,26 @@
  */
 package org.elasticsearch.search.aggregations;
 
+import static org.elasticsearch.test.InternalAggregationTestCase.DEFAULT_MAX_BUCKETS;
```

Comment:
I think your import order config is a bit confused here. I believe we usually put static last. Not a big deal, but it makes the diff bigger.